docker-compose.yaml

```diff
@@ -59,7 +59,7 @@ services:
       - '${AGENT_CONFIG_PATH_LOCAL}:${AGENT_CONFIG_PATH}'
       - 'demo_demo_logs:${AGENT_LOGS_PATH}'
     entrypoint:
-      - '/bin/grafana-agent'
+      - '/bin/agent'
```

Comment:
Q: A few weeks ago we merged a PR because in upstream symbolic links are now prefixed with `grafana-*`.
Has this chnaged again? 

* [PR: docker-compose: fix grafana-agent path #173 ](https://github.com/grafana/faro-web-sdk/pull/173)
* [Docs/Upgrade guide v0.33](https://grafana.com/docs/agent/latest/static/upgrade-guide/#v033)

Reply:
Yes, this is new. I think I had something misconfigured on my end and this fixed things. Will revert this change.